libsignal/src/main/java/org/session/libsignal/protos/SignalProtos.java

```diff
@@ -101,7 +101,7 @@ private ServerCertificate(
             e.getMessage()).setUnfinishedMessage(this);
       } finally {
         this.unknownFields = unknownFields.build();
-        makeExtensionsImmutable();
+        //makeExtensionsImmutable(); // ACL - no longer available after protobuf-java update?!
```

Comment:
Is it safe to have all these methods commented out?

Reply:
What I'd immediately missed due to code folding of headers when looking at `SignalProtos.java` was the first line: 
`// Generated by the protocol buffer compiler.  DO NOT EDIT!`

So, I have a feeling that this file should be regenerated (which will undo my manual edit). It looks like we're not regenerating the protobuf files on build, which we probably should be - will look into it now.

Reply:
@AL-Session on iOS the protobuf files are build with a makefile - taking a quick look there is one in `libsession-util/libsession-util/proto` so you might be able to just run `make` in that directory 🤷

Reply:
@simophin This is the file that looks like it needs to be regenerated via protoc

Reply:
> @AL-Session on iOS the protobuf files are build with a makefile - taking a quick look there is one in `libsession-util/libsession-util/proto` so you might be able to just run `make` in that directory 🤷

Aha! Thank you - I'll take a look at that now :)

Reply:
This file actually isn't used so has been deleted (also, the .proto files we _do_ use have been regenerated using protobuf 3.25.3).